extensions/skyportal/static/js/components/SourceMobile.jsx

```diff
@@ -292,34 +296,6 @@ const SourceMobile = WidthProvider(
                       </div>
                     </div>
                   </div>
-                  {source.duplicates && (
```

Comment:
@jadalilleboe Was this removed by accident perchance?

Reply:
yes I just copy pasted the component from skyportal

Reply:
I didn't realize there was fritz specific stuff in there